pkg/store/github/github_test.go

```diff
@@ -74,7 +75,7 @@ func TestGithubStore(t *testing.T) {
 		}
 
 		if ghErr.Message != "github went belly up or something" {
-			t.Errorf("user err is %s, want 'github went belly up or something'", err.Error())
+			t.Errorf("user err is %s, result 'github went belly up or something'", err.Error())
```

Comment:
I don't understand this change

Reply:
Its an unintended side effect from a time when I refactored a test.
Fixed.

Reply:
regression :D